test/common/index.js

```diff
@@ -261,13 +261,16 @@ function platformTimeout(ms) {
   return ms; // ARMv8+
 }
 
+const performance = globalThis.performance;
+
 let knownGlobals = [
   atob,
   btoa,
   clearImmediate,
   clearInterval,
   clearTimeout,
   global,
+  performance,
```

Comment:
https://github.com/nodejs/node/blob/d1e2184c8ea0362ad695d51fb6409fc1741a227b/test/common/index.js#L277-L284

I think this should the same trick (pushing to known globals only after testing if it exists) – or we should always add `AbortController` to the list without condition, as long as we are consistent.